policyengine_us/parameters/gov/states/ky/tax/income/exemptions/homestead/amount.yaml

```diff
@@ -0,0 +1,17 @@
+description: Kentucky provides this homestead exemption amount.
+values:
+  2021-01-01: 40_500
+  2023-01-01: 46_350
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Kentucky homestead exemption amount
+  reference:
+  - title: Kentucky Department of Revenue - Homestead Exemption
+    href: https://revenue.ky.gov/Property/Residential-Farm-Commercial-Property/Pages/Homestead-Exemption.aspx
+  - title: Kentucky Homestead exemption - Application - Qualification
+    href: https://casetext.com/statute/kentucky-revised-statutes/title-11-revenue-and-taxation/chapter-132-levy-and-assessment-of-property-taxes/section-132810-homestead-exemption-application-qualification
+    # Law does not provide inflation adjusted amounts
+  - title: Kentucky Department of Revenue sets 2023-2024 Homestead Exemption
+    href: https://www.kentucky.gov/Pages/Activity-stream.aspx?n=FinanceandAdministrationCabinet&prId=210
```

Comment:
We can use this reference, which has the inflation-adjusted exemption amounts:
https://revenue.ky.gov/Property/Residential-Farm-Commercial-Property/Pages/HomesteadExemptionHistoricalData.aspx